tests/test_search.py

```diff
@@ -250,9 +252,8 @@ def test_search_date_string(self, shortname, datetime_string):
               settings={'RELATIVE_BASE': datetime.datetime(2000, 1, 1)}),
 
         # Dutch
-        param('nl', ' De meest dramatische uitbreiding van het conflict vond plaats op 22 juni 1941 met de '
```

Comment:
Updated the text as it contains `CET` which is a valid timezone and causes a false positive.

Reply:
Let's try a little harder to make these tests work without modifying, OK?

Reply:
I reverted the test case and now it passed.